### ISSUE TITLE
Unable to process graphs with <img> tags

### ISSUE THREAD

**Issue by atextor** [nidi3/graphviz-java#98]
When including a table in an HTML label, it is possible in Graphviz to refer to external images using `<img>` tags there. I have been unable to get a graph using this feature processed by graphviz-java. When using the GraphvizCmdLineEngine, a temporary directory is created with no API hooks to copy resources there (please correct me if I'm wrong), causing the dot binary to not find the image file. With the other Engines, I just get Errors about invalid labels.
Here is a minimal example:

minimal.dot:
```
digraph G {
 Foo [label=<
  <table>
    <tr>
      <td><img src="foo.png" /></td>
    </tr>
  </table> >]
}
```

Java:
```java
Graphviz.fromFile(new File("minimal.dot")).render(Format.PNG).toFile(...);
```

**Comment by nidi3** [comment]
Images in HTML labels are not supported by viz.js: https://github.com/mdaines/viz.js/issues/125
What works is absolute paths with the `GraphvizCmdLineEngine`
What I'm goint to support is 
- relative paths for `GraphvizCmdLineEngine`
- the `image` attribute for nodes 

hope that helps

**Comment by nidi3** [comment]
#101 is fixed and available in 0.8.2, updated README.

**Comment by atextor** [comment]
Awesome, thank you!
